app/helpers/assignments_helper.rb

```diff
@@ -1,2 +1,47 @@
 module AssignmentsHelper
+	MAX = 26**4-1; # the maximum number that can be hashed to
+    CODESTRING = "ydlgknmzxjbctfiaqsrwoevuhp"; # random ordering of the alphabet
```

Comment:
why is line 3 on a totally different indentation level than line 2?

Reply:
That's not appearing on my end. Very strange.